packages/@aws-cdk/aws-apigateway/test/integ.stepfunctions-api.js.snapshot/StepFunctionsRestApiDeploymentStack.assets.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "30.0.0",
+  "version": "30.1.0",
```

Comment:
why are these changing? There should be no snapshot changes for this change.

Reply:
You are right.
There is no difference in the snapshot, so the change was not originally necessary. I will revert to the original state.

Reply:
Ah. I understand.
So this.role is allowed iam.IRole, so there could be side effects if, for example, fromLookup references an existing IAM role?

I would like to incorporate your SUGGESTION.